### PR TITLE
[DX-1437] optimise --> optimize

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by Roeegg2** [comment]
There were a few things I'm not sure about. Let me know if they need to be changed:

1. Didn't touch any of the JSON and conf examples (files `/tyk-docs/tyk-docs/content/basic-config-and-security/reduce-latency/caching/optimise-cache.md`, `/tyk-docs/tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways.md`, `/tyk-docs/tyk-docs/content/planning-for-production/redis.md`, `/tyk-docs/tyk-docs/content/tyk-stack/tyk-gateway/configuration/redis-sentinel.md`)
2. Didn't change the anchors (such as in files `/tyk-docs/tyk-docs/content/developer-support/frequently-asked-questions/how-to-reduce-cpu-usage-in-a-redis-cluster.md`) and the titles (`/tyk-docs/tyk-docs/content/shared/mdcb-config.md`)

**Review by letzya — APPROVED** [review]

File: tyk-docs/content/developer-support/frequently-asked-questions/how-to-reduce-cpu-usage-in-a-redis-cluster.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ Ensure that the `addrs` array is populated in the [Tyk Gateway]({{< ref "tyk-oss
...
+1. **Optimize Analytics Configuration**: In the Tyk Gateway configuration (tyk.conf), set [analytics_config.enable_multiple_analytics_keys]({{< ref "tyk-oss-gateway/configuration#analytics_configenable_multiple_analytics_keys" >}}) to true. This option allows Tyk to distribute analytics data across Redis nodes, using multiple keys for the analytics. There's a corresponding option for Self Managed MDCB, also named [enable_multiple_analytics_keys]({{< ref "tyk-multi-data-centre/mdcb-configuration-options#enable_multiple_analytics_keys" >}}). Useful only if the gateways in the data plane are configured to send analytics to MDCB.
+2. **Optimize Connection Pool Settings**: Adjust the [optimisation_max_idle]({{< ref "tyk-oss-gateway/configuration#storageoptimisation_max_idle" >}}) and [optimisation_max_active]({{< ref "tyk-oss-gateway/configuration#storageoptimisation_max_active" >}}) settings in the configuration files to ensure that the connection pool can handle the analytics workload without overloading any Redis shard.
```
Comment (letzya) [inline tyk-docs/content/developer-support/frequently-asked-questions/how-to-reduce-cpu-usage-in-a-redis-cluster.md]:
```suggestion
2. **Optimize Connection Pool Settings**: Adjust the [optimization_max_idle]({{< ref "tyk-oss-gateway/configuration#storageoptimisation_max_idle" >}}) and [optimization_max_active]({{< ref "tyk-oss-gateway/configuration#storageoptimisation_max_active" >}}) settings in the configuration files to ensure that the connection pool can handle the analytics workload without overloading any Redis shard.
```

Comment (letzya) [reply tyk-docs/content/developer-support/frequently-asked-questions/how-to-reduce-cpu-usage-in-a-redis-cluster.md]: @Roeegg2 FYI - change the label

**Review by letzya — APPROVED** [review]

**Comment by letzya** [comment]
> There were a few things I'm not sure about. Let me know if they need to be changed:
> 
> 1. Didn't touch any of the JSON and conf examples (files `/tyk-docs/tyk-docs/content/basic-config-and-security/reduce-latency/caching/optimise-cache.md`, `/tyk-docs/tyk-docs/content/tyk-cloud/environments-deployments/hybrid-gateways.md`, `/tyk-docs/tyk-docs/content/planning-for-production/redis.md`, `/tyk-docs/tyk-docs/content/tyk-stack/tyk-gateway/configuration/redis-sentinel.md`)
> 2. Didn't change the anchors (such as in files `/tyk-docs/tyk-docs/content/developer-support/frequently-asked-questions/how-to-reduce-cpu-usage-in-a-redis-cluster.md`) and the titles (`/tyk-docs/tyk-docs/content/shared/mdcb-config.md`)

- in how-to-reduce-cpu-usage-in-a-redis-cluster.md - Fixed the labels to links but not the links
- Rest - no need to change

**Pull request merged**